drivers/input/input_analog_axis.c

```diff
@@ -221,8 +238,44 @@ static int analog_axis_init(const struct device *dev)
 
 	k_thread_name_set(&data->thread, dev->name);
 
+	k_timer_start(&data->timer,
+		      K_MSEC(cfg->poll_period_ms), K_MSEC(cfg->poll_period_ms));
+
+	ret = pm_device_runtime_enable(dev);
+	if (ret < 0) {
+		LOG_ERR("Failed to enable runtime power management");
+		return ret;
+	}
```

Comment:
this will start the timer, to then stop it, you may optimize this when PM is enabled (by not calling `k_timer_start`, and using `pm_device_init_suspended`).

Reply:
Done, not sure I like the code too much though to be honest, an extra conditional and moving the cfg variable mid body. Meh guess it's fine, you tell me.